src/sys/windows/selector.rs

```diff
@@ -405,10 +404,8 @@ impl Selector {
 
 #[derive(Debug)]
 pub struct SelectorInner {
-    lock: Mutex<()>,
     cp: Arc<CompletionPort>,
-    active_poll_count: UnsafeCell<usize>,
```

Comment:
Isn't this required for correctness?

Reply:
No tests failed, so if it is we don't have sufficient test coverage to show that.

Reply:
According to @piscisaureus this is ok since we use `&mut self` in `Poll::poll`, see https://github.com/tokio-rs/mio/pull/1112#issuecomment-549620142